include/NovelRT/Graphics/FontSet.h

```diff
@@ -1,7 +1,7 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 #ifndef NOVELRT_GRAPHICS_FONTSET_H
-
+#define NOVELRT_GRAPHICS_FONTSET_H
 #ifndef NOVELRT_H
 #error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
 #endif
```

Comment:
We're still using the centralised header for this? Or was this not needed for pt. 1?

Reply:
This is the legacy graphics namespace. Updates will not be made to this namespace as it is getting nuked from orbit in the future once everything is in place.